spec/acceptance/class_spec.rb

```diff
@@ -37,6 +37,19 @@ class { 'erlang': epel_enable => true}
       it { is_expected.to be_enabled }
       it { is_expected.to be_running }
     end
+
+    describe file('/etc/rabbitmq') do
+      it { is_expected.to be_directory }
+      it { is_expected.to be_owned_by 'rabbitmq' }
+      it { is_expected.to be_grouped_into 'rabbitmq' }
+    end
```

Comment:
_think_ this is the right place for these to live?
seems sensible to test these in the acceptance tests one way or another, I think, right?